src/backend/vulkan/src/command.rs

```diff
@@ -836,30 +836,42 @@ impl com::CommandBuffer<Backend> for CommandBuffer {
             .cmd_draw_indexed_indirect(self.raw, buffer.raw, offset, draw_count, stride)
     }
 
-    unsafe fn draw_mesh_tasks(&mut self, _: TaskCount, _: TaskCount) {
-        unimplemented!()
+    unsafe fn draw_mesh_tasks(&mut self, task_count: TaskCount, first_task: TaskCount) {
+        if let Some(mesh_fn) = &self.device.mesh_fn {
```

Comment:
let's just do `expect` with this message

Reply:
Done.